test/test_browser.py

```diff
@@ -2992,6 +2992,48 @@ def test_sdl2_image_formats(self):
       '--use-port=sdl2', '--use-port=sdl2_image:formats=jpg',
     ])
 
+  @requires_graphics_hardware
+  @also_with_wasm2js
```

Comment:
```suggestion
```
Perhaps we don't need wasm2js here, as there isn't something very specific to test in SDL3+wasm2js (vs SDL2+wasm2js).